app/state.go

```diff
@@ -1673,6 +1673,47 @@ func (rs *updateRollbackState) Handle(ctx *StateContext, c Controller) (State, b
 			return rs.HandleError(ctx, c, NewFatalError(err))
 		}
 	}
+
+	// Query the rollback parameter from the Update Module, in case it has
+	// not been called previously (MEN-4882)
+	for n, i := range c.GetInstallers() {
+		v, err := rs.Update().RebootRequested.Get(n)
+		if err != nil {
+			log.Errorf(
```

Comment:
How about dialling this back to `Debugf`? After this fix it's not a bug/error anymore, so I don't think it's necessary to be loud about it.